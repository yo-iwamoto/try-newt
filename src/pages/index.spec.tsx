import Page from './index.page';
import { renderNextPage } from '@/__tests__/util';

describe('[page] /', () => {
  it('renders / unchanged', () => {
    const { container } = renderNextPage(<Page />);

    expect(container).toMatchSnapshot();
  });
});
