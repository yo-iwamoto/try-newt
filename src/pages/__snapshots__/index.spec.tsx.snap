// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[page] / renders / unchanged 1`] = `
<div>
  <div>
    <h1>
      Hello world!
    </h1>
  </div>
</div>
`;
